platform/testbed/src/integrationTest/resources/admin/snippet/create_get/01.lua

```diff
@@ -0,0 +1,21 @@
+sys = require('sys')
+
+err, create_snippet_req = sys.snippet.create({
+    name = 'test-snippet',
+    inputs = {},
+    value = [=[40 + 2]=]
+})
+
+sys.await_completed(create_snippet_req)
+assert(err == nil)
```

Comment:
```suggestion
assert(err == nil)
sys.await_completed(create_snippet_req)
```

I did this wrong in the past myself - we want to make sure that err == ni before we are accessing create_snippet_req. Reason being - create_snippet_req is nil in the event of an error --> null pointer exception